code/modules/reagents/reagent_containers/iv_bag.dm

```diff
@@ -195,6 +198,16 @@
 /obj/item/reagent_containers/iv_bag/blood/OMinus
 	blood_type = "O-"
 
+/obj/item/reagent_containers/iv_bag/blood/vox
+	blood_type = "O-"
+	blood_species = "Vox"
+	blood_color = "#2299FC"
+	one_species_only = TRUE
+
+/obj/item/reagent_containers/iv_bag/blood/vox/Initialize(mapload)
+	. = ..()
+	name = "[initial(name)] - O- Vox Blood"
```

Comment:
I suggest calling it just "Vox Blood" similarly to how we have "Slime Jelly". If there are no separate blood types (I don't think vox need more than one), denoting their type will just mislead people.